app/views/comments/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1><%= link_to @post.title, @post.link %></h1>
+
+<%= link_to 'New Comment', new_post_comment_path(@post), class: 'btn btn-primary' %>
+
+<% @post.comment.each do |comment| %>
+<div class="well">
+  <p><%= comment.content %></p>
+  <strong>By <%= comment.user.name %></strong>
+ <!--  <%= render partial: 'comments/voting', locals: {post: post} %> -->
```

Comment:
Finally, let's change line 9 to the following:
```
 <%= render partial: 'comments/voting', locals: {post: @post, comment: comment} %>
```
The key is passing in @post and comment so that the partial knows what it's dealing with.